qemu/tests/cfg/live_snapshot.cfg

```diff
@@ -42,3 +42,28 @@
             dd_cmd = "dd if=/dev/zero of=%s bs=4K count=1024 oflag=dsync"
```

Comment:
@CongLi , I guess better to add no windows here first, since yfu plan to add this to acceptance loop.